lib/player.js

```diff
@@ -4409,6 +4407,35 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       }
     }
 
+    // Deprecate 'streaming.forceHTTP' configuration.
+    if (config['streaming'] && 'forceHTTP' in config['streaming']) {
+      shaka.Deprecate.deprecateFeature(5,
+          'streaming.forceHTTP configuration',
+          'Please Use networking.forceHTTP instead.');
+      config['networking']['forceHTTP'] = config['streaming']['forceHTTP'];
+      delete config['streaming']['forceHTTP'];
+    }
+
+    // Deprecate 'streaming.forceHTTPS' configuration.
+    if (config['streaming'] && 'forceHTTPS' in config['streaming']) {
+      shaka.Deprecate.deprecateFeature(5,
+          'streaming.forceHTTPS configuration',
+          'Please Use networking.forceHTTP instead.');
+      config['networking']['forceHTTPS'] = config['streaming']['forceHTTPS'];
+      delete config['streaming']['forceHTTPS'];
+    }
+
+    // Deprecate 'streaming.minBytesForProgressEvents' configuration.
+    if (config['streaming'] &&
+        'minBytesForProgressEvents' in config['streaming']) {
+      shaka.Deprecate.deprecateFeature(5,
```

Comment:
Thank you for taking care of the backward compatibility and transition to the new API.